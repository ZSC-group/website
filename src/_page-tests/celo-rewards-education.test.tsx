import CeloRewardsEducation from 'pages/celo-rewards-education'
import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { TestProvider } from 'src/_page-tests/test-utils'

describe('CeloRewardsEducation', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <TestProvider>
          <CeloRewardsEducation />
        </TestProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
