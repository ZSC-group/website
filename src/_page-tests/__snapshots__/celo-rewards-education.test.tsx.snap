// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CeloRewardsEducation renders 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "marginTop": "20px",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 0,
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "flexGrow": 0,
          "flexShrink": 0,
          "msFlexNegative": 0,
          "msFlexPositive": 0,
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <h1
        aria-level="1"
        className="css-reset-4rbku5 css-text-901oao"
        dir="auto"
        onClick={[Function]}
        role="heading"
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "26px",
            "fontWeight": "500",
            "lineHeight": "32px",
            "marginBottom": "16px",
            "textRendering": "geometricPrecision",
            "textTransform": "none",
          }
        }
      >
        CELO Rewards Program
      </h1>
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "textRendering": "geometricPrecision",
          }
        }
      >
        Confirmed cUSD wallets can earn rewards by holding a minimum monthly average balance of:
        

        <ul
          className="css-reset-4rbku5 css-view-1dbjc4n"
          role="list"
          style={
            Object {
              "display": "inline-flex",
              "listStyle": "disc",
              "marginTop": "20px",
              "paddingLeft": "20px",
            }
          }
        >
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                20 cUSD
              </span>
              

              Earn $1 worth of CELO per month
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                100 cUSD
              </span>
              

              Earn $5 worth of CELO per month
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                500 cUSD
              </span>
              

              Earn $10 worth of CELO per month
            </span>
          </li>
        </ul>
        

        Rewards are credited to your wallet’s CELO balance at the beginning of each month until the program ends.
        


        Participate with any confirmed or verified cUSD account by holding a minimum average monthly balance of 20.00 cUSD during the earnings period.
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 0,
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "flexGrow": 0,
          "flexShrink": 0,
          "msFlexNegative": 0,
          "msFlexPositive": 0,
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "textRendering": "geometricPrecision",
          }
        }
      >
        <span
          className="css-text-901oao css-textHasAncestor-16my406"
          onClick={[Function]}
          style={
            Object {
              "color": "rgba(46,51,56,1.00)",
              "fontFamily": "Jost, futura-pt, futura, sans-serif",
              "fontSize": "18px",
              "fontWeight": "bold",
              "lineHeight": "24px",
              "textRendering": "geometricPrecision",
            }
          }
        >
          Rewards Kickoff:
        </span>
         
        CELO rewards are 4x until March 1st! Expect to receive weekly rewards.
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 0,
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "flexGrow": 0,
          "flexShrink": 0,
          "msFlexNegative": 0,
          "msFlexPositive": 0,
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <h1
        aria-level="1"
        className="css-reset-4rbku5 css-text-901oao"
        dir="auto"
        onClick={[Function]}
        role="heading"
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "22px",
            "fontWeight": "500",
            "lineHeight": "26px",
            "marginBottom": "16px",
            "textRendering": "geometricPrecision",
            "textTransform": "none",
          }
        }
      >
        Here’s how it works:
      </h1>
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "textRendering": "geometricPrecision",
          }
        }
      >
        1) Transfer cUSD to your wallet to participate! You can find the instructions for adding cUSD to a Valora wallet 
        <div
          className="css-view-1dbjc4n"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          style={
            Object {
              "display": "inline",
            }
          }
        >
          <a
            className="css-reset-4rbku5 css-cursor-18t94o4 css-text-901oao css-textHasAncestor-16my406"
            data-focusable={true}
            href="https://valoraapp.com/support/fund-wallet"
            onClick={[Function]}
            rel=" noopener noreferrer"
            role="link"
            style={
              Object {
                "WebkitTextDecorationStyle": "solid",
                "color": "rgba(26,183,117,1.00)",
                "cursor": "pointer",
                "display": "inline",
                "opacity": 1,
                "textDecoration": "none",
                "textDecorationStyle": "solid",
              }
            }
            target="_blank"
          >
            here
          </a>
        </div>
        .

2) Confirm your wallet. An easy way to do this is to confirm your phone number with the Valora app by following the onboarding steps in the app!

3) Rewards are calculated and distributed monthly depending on how much cUSD you have held in your wallet:
        

        <ul
          className="css-reset-4rbku5 css-view-1dbjc4n"
          role="list"
          style={
            Object {
              "display": "inline-flex",
              "listStyle": "disc",
              "marginTop": "20px",
              "paddingLeft": "20px",
            }
          }
        >
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 1
              </span>
              

              Minimum monthly average balance of 20 cUSD → earn $1 worth of CELO per month
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 2
              </span>
              

              Minimum monthly average balance of 100 cUSD → earn $5 worth of CELO per month
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 3
              </span>
              

              Minimum monthly average balance of 500 cUSD → earn $10 worth of CELO per month
            </span>
          </li>
        </ul>
        

        4) Rewards will be credited to your CELO balance at the end of the earnings period (expected monthly).
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 0,
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "flexGrow": 0,
          "flexShrink": 0,
          "msFlexNegative": 0,
          "msFlexPositive": 0,
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <h1
        aria-level="1"
        className="css-reset-4rbku5 css-text-901oao"
        dir="auto"
        onClick={[Function]}
        role="heading"
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "22px",
            "fontWeight": "500",
            "lineHeight": "26px",
            "marginBottom": "16px",
            "textRendering": "geometricPrecision",
            "textTransform": "none",
          }
        }
      >
        What is CELO vs cUSD?
      </h1>
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "textRendering": "geometricPrecision",
          }
        }
      >
        CELO and cUSD are the current assets supported by wallets like Valora on the Celo Network, which provides fast, secure, and stable digital payments. The two main differences between the two assets are function and price.

Celo Dollars, or cUSD, is a stablecoin whose value tracks that of the US Dollar. Since it tracks the US Dollar, cUSD is said to be ‘stable’. It is also built for accessibility and ease of use, so you can easily pay, save and send with cUSD using Valora.

CELO is the native asset and core utility, governance, reserve, and staking mechanism for the Celo Network. When you own CELO, you can have a say by voting on how the Celo Network evolves and grows over time. CELO has a fixed supply, and a variable value related to the total value of stablecoins in circulation in the Celo ecosystem. 
        <div
          className="css-view-1dbjc4n"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          style={
            Object {
              "display": "inline",
            }
          }
        >
          <a
            className="css-reset-4rbku5 css-cursor-18t94o4 css-text-901oao css-textHasAncestor-16my406"
            data-focusable={true}
            href="https://medium.com/celoorg/celo-gold-holders-make-your-voice-heard-through-on-chain-governance-96cb5a1e8b90"
            onClick={[Function]}
            rel=" noopener noreferrer"
            role="link"
            style={
              Object {
                "WebkitTextDecorationStyle": "solid",
                "color": "rgba(26,183,117,1.00)",
                "cursor": "pointer",
                "display": "inline",
                "opacity": 1,
                "textDecoration": "none",
                "textDecorationStyle": "solid",
              }
            }
            target="_blank"
          >
            Learn more about governance on the Celo Network.
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 0,
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "flexGrow": 0,
          "flexShrink": 0,
          "msFlexNegative": 0,
          "msFlexPositive": 0,
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <h1
        aria-level="1"
        className="css-reset-4rbku5 css-text-901oao"
        dir="auto"
        onClick={[Function]}
        role="heading"
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "26px",
            "fontWeight": "500",
            "lineHeight": "32px",
            "marginBottom": "16px",
            "textRendering": "geometricPrecision",
            "textTransform": "none",
          }
        }
      >
        Terms and Conditions
      </h1>
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "textRendering": "geometricPrecision",
          }
        }
      >
        When you hold a qualifying cUSD balance at your confirmed address, you may be eligible for a reward of CELO. Rewards will be distributed at the end of the earnings period, typically monthly, on a first come, first serve basis (by wallet phone number confirmation date) up to a total limit of 20,000 CELO per calendar month. During the introduction period, this total limit will be brought up to 20,000 CELO per week. The introduction period is 20:00 UTC January 25th, 2021 through 19:59 UTC March 1, 2021. Monthly earning periods will begin on the first of each month, at 20:00 UTC and end on the first of each month, at 19:59 UTC, until the program stops or reaches the total limit of 200,000 CELO. During the initial introduction phase, weekly earning periods will begin on the Mondays, at 20:00 UTC and end the following Monday, at 19:59 UTC, until the program stops reaches the total limit of 200,000 CELO. The first earning period is expected to start on January 25th, 2021, at 20:00 UTC.

A confirmed address must have completed 3 text message attestations to be eligible for this program. You must have an eligible address to participate in the CELO Rewards program. This program may be terminated at any time and is void where prohibited.

Rewards are calculated and distributed monthly (or weekly, during the introduction period) depending on how much cUSD you have held in your wallet:
        

        <ul
          className="css-reset-4rbku5 css-view-1dbjc4n"
          role="list"
          style={
            Object {
              "display": "inline-flex",
              "listStyle": "disc",
              "marginTop": "20px",
              "paddingLeft": "20px",
            }
          }
        >
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 1
              </span>
              

              Minimum monthly (weekly, during the introduction period) average balance of 20 cUSD → earn $1 worth of CELO per month (per week during the introduction period)
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 2
              </span>
              

              Minimum monthly (weekly, during the introduction period) average balance of 100 cUSD → earn $5 worth of CELO per month (per week during the introduction period)
            </span>
          </li>
          <li
            className="css-reset-4rbku5 css-text-901oao css-textHasAncestor-16my406"
            onClick={[Function]}
            role="listitem"
            style={
              Object {
                "color": "rgba(46,51,56,1.00)",
                "display": "list-item",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "listStyle": "inherit",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              onClick={[Function]}
              style={
                Object {
                  "color": "rgba(46,51,56,1.00)",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "18px",
                  "fontWeight": "400",
                  "lineHeight": "24px",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406"
                onClick={[Function]}
                style={
                  Object {
                    "color": "rgba(46,51,56,1.00)",
                    "fontFamily": "Jost, futura-pt, futura, sans-serif",
                    "fontSize": "18px",
                    "fontWeight": "bold",
                    "lineHeight": "24px",
                    "textRendering": "geometricPrecision",
                  }
                }
              >
                Tier 3
              </span>
              

              Minimum monthly (weekly, during the introduction period) average balance of 500 cUSD → earn $10 worth of CELO per month (per week during the introduction period)
            </span>
          </li>
        </ul>
        

        The first earning period is expected to start on January 25th, 2021, at 20:00 UTC, and will continue to be distributed on a weekly basis during the introduction period, then monthly, for the duration of the program. 

The balance calculation is determined by averaging the cUSD balance over all blocks in the blockchain during the earning period for each confirmed address. 

CELO price will be valued based on the average “Close” price for CELO as indicated on coinmarketcap.com for the earning period. For instance, for the first week of February, CELO price will be the sum of the 7 days Close price of CELO on coinmarketcap.com from February 1st, 2021 (included) to February 8th, 2021 (excluded), divided by 7. 

The reward rate and terms are subject to change over time, and are void where prohibited. Rewards are not guaranteed, may be discontinued at any time, and will be allocated on a best efforts basis. Actual results may vary, based on various factors such as leap years, timing of transfers, rounding, fund exhaustion, and variation in reward rates.
      </div>
    </div>
  </div>
  <a
    className="css-reset-4rbku5 css-cursor-18t94o4 css-text-901oao"
    data-focusable={true}
    data-testid="AddCusd"
    dir="auto"
    href="celo://wallet/cashIn"
    onClick={[Function]}
    role="link"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "WebkitBoxPack": "center",
        "WebkitJustifyContent": "center",
        "alignItems": "center",
        "backgroundColor": "rgba(26,183,117,1.00)",
        "borderBottomLeftRadius": "24px",
        "borderBottomRightRadius": "24px",
        "borderBottomWidth": "0px",
        "borderLeftWidth": "0px",
        "borderRightWidth": "0px",
        "borderTopLeftRadius": "24px",
        "borderTopRightRadius": "24px",
        "borderTopWidth": "0px",
        "color": "rgba(255,255,255,1.00)",
        "cursor": "pointer",
        "display": "flex",
        "fontFamily": "Jost, futura-pt, futura, sans-serif",
        "fontSize": "17px",
        "fontWeight": "600",
        "height": "48px",
        "justifyContent": "center",
        "lineHeight": "16px",
        "marginBottom": "30px",
        "marginLeft": "20px",
        "marginRight": "20px",
        "marginTop": "30px",
        "msFlexAlign": "center",
        "msFlexPack": "center",
        "textAlign": "center",
        "textRendering": "geometricPrecision",
      }
    }
  >
    Add cUSD
  </a>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxFlex": 0,
          "WebkitBoxPack": "center",
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "WebkitJustifyContent": "center",
          "alignItems": "center",
          "display": "flex",
          "flexGrow": 0,
          "flexShrink": 0,
          "justifyContent": "center",
          "msFlexAlign": "center",
          "msFlexNegative": 0,
          "msFlexPack": "center",
          "msFlexPositive": 0,
          "paddingBottom": "0px",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "0px",
          "width": "100%",
        }
      }
    >
      <h1
        aria-level="1"
        className="css-reset-4rbku5 css-text-901oao"
        dir="auto"
        onClick={[Function]}
        role="heading"
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "22px",
            "fontWeight": "500",
            "lineHeight": "26px",
            "marginBottom": "4px",
            "textRendering": "geometricPrecision",
            "textTransform": "none",
          }
        }
      >
        Questions?
      </h1>
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "color": "rgba(46,51,56,1.00)",
            "fontFamily": "Jost, futura-pt, futura, sans-serif",
            "fontSize": "18px",
            "fontWeight": "400",
            "lineHeight": "24px",
            "marginBottom": "40px",
            "textAlign": "center",
            "textRendering": "geometricPrecision",
          }
        }
      >
        Send us a message at

        <div
          className="css-view-1dbjc4n"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          style={
            Object {
              "display": "inline",
            }
          }
        >
          <a
            className="css-reset-4rbku5 css-cursor-18t94o4 css-text-901oao css-textHasAncestor-16my406"
            data-focusable={true}
            href="mailto:support@valoraapp.com"
            onClick={[Function]}
            role="link"
            style={
              Object {
                "WebkitTextDecorationStyle": "solid",
                "color": "rgba(26,183,117,1.00)",
                "cursor": "pointer",
                "display": "inline",
                "opacity": 1,
                "textDecoration": "none",
                "textDecorationStyle": "solid",
              }
            }
          >
            support@valoraapp.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
